eth/backend.go

```diff
@@ -220,6 +221,19 @@ func New(stack *node.Node, config *Config) (*Ethereum, error) {
 	stack.RegisterAPIs(eth.APIs())
 	stack.RegisterProtocols(eth.Protocols())
 	stack.RegisterLifecycle(eth)
+	// Check for unclean shutdown
+	if uncleanShutdowns, discards, err := rawdb.UpdateUncleanShutdownMarker(chainDb); err != nil {
+		log.Error("Could not update unclean-shutdown-marker list", "error", err)
+	} else {
+		if discards > 0 {
+			log.Warn("Old unclean shutdowns found", "count", discards)
+		}
+		for _, tstamp := range uncleanShutdowns {
```

Comment:
It's inaccurate actually. It's the timestart of the START, not the CRASH. Usually geth will run for a very long time

Reply:
Can we have an underlying thread to update the shutdown marker e.g. every 1 minute? For the cost wise, it's basically no overhead but at least we can decrease the bias

Reply:
Maybe we could do that in a follow-up PR? 
For now I can replace `time` with `booted`, to clarify that it's the startup timestamp, not the crash timestamp